src/erlcloud_ec2.erl

```diff
@@ -2936,6 +2946,168 @@ net_if_params(#ec2_net_if{}=X) ->
 net_if_params(List, Prefix) ->
     erlcloud_aws:param_list([net_if_params(X) || X <- List], Prefix).
 
+%%
+%% Function for making calls to CreateFlowLogs action
+%% DescribeFlowLogs Documentation: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateFlowLogs.html
+%%
+-spec create_flow_logs(
+        string(),
+        vpc | subnet | network_interface,
+        [string()],
+        accept | reject | all,
+        string()) -> ok_error(string()).
+create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn) ->
+    create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, none, default_config()).
+
+-spec create_flow_logs(
+        string(),
+        vpc | subnet | network_interface,
+        [string()],
+        accept | reject | all,
+        string(),
+        string() | aws_config()) -> ok_error(string()).
+create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, Config)
+    when is_record(Config, aws_config) ->
+    create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, none, Config);
+create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, ClientToken) ->
+    create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, ClientToken, default_config()).
+
+-spec create_flow_logs(
+        string(),
+        vpc | subnet | network_interface,
+        [string()],
+        accept | reject | all,
+        string(),
+        string(),
+        aws_config()) -> ok_error(string()).
+create_flow_logs(LogGroupName, ResourceType, ResourceIDs, TrafficType, DeliverLogsPermissionArn, ClientToken, Config)
+    when is_record(Config, aws_config) ->
+    {Resources, _} = lists:foldl(fun(ID, {Acc, Index}) ->
+                                    I = integer_to_list(Index),
+                                    ResourceID = "ResourceId."++I,
+                                    {[{ResourceID, ID} | Acc], Index+1}
+                            end, {[], 1}, ResourceIDs),
+    RT = case ResourceType of
+            vpc -> "VPC";
+            subnet -> "Subnet";
+            network_interface -> "NetworkInterface"
+        end,
+    TT = case TrafficType of
+            accept -> "ACCEPT";
+            reject -> "REJECT";
+            all -> "ALL"
+        end,
+    P = [
+        {"LogGroupName", LogGroupName},
+        {"ResourceType", RT},
+        {"TrafficType", TT},
+        {"DeliverLogsPermissionArn", DeliverLogsPermissionArn}
+    ] ++ Resources,
+    Params = case ClientToken of
+                none -> P;
+                _ -> P ++ {"ClientToken", ClientToken}
+            end,
+    case ec2_query(Config, "CreateFlowLogs", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok, [get_text(Item) || Item <- xmerl_xpath:string("/CreateFlowLogsResponse/flowLogIdSet/item", Doc)]};
```

Comment:
it seems the API supports partial success as it also has `unsuccessful` item in response. please add that. 
same applies to `clientToken`